src/routes/web.php

```diff
@@ -14,5 +14,5 @@
 */
 
 Route::get('/', function () {
-    return view('welcome');
+    return view('login');
 });
```

Comment:
Create a separate file for the controller.

Reply:
I have resolved this comment, sir. Thank you!